### DISCUSSION TITLE
animating georaster on DEM

### DISCUSSION THREAD

**Discussion by alberoman** [domlysz/BlenderGIS#465] [Q&A]
Hi, and first of all thanks for this add-on that is really super cool and well done. 
I have plenty of georaster data that correspond to different times that I would like to plot on a DEM and to animate (so that  specific raster should be plotted at specific frames). I wrote a script in which there is a function calling the operator importgis.georaster, and this function is called by bpy.app.handlers.frame_change_post in order to control the frame position.
So that at each frame basically georaster() is called adding a uv texture to the material (which is then applied by the function to the mesh of the DEM). The problem with this is that there is a maximum of 8 UV slots so that basically after 8 frames there are are no more slots available. I tried deleting the materials and the UV texture after each frame change but this creates some unexpected Blender crash. Do you know if it is possible somehow to do this animation integrating BlenderGIS?
Again congrats for this great work!

**Comment by s-leger** [comment]
Would rather load all images into a single animated image sequance (same texture node but with more than one image as input). May probably require proper naming of texture files like lat-lon-time in order to work, but this way you avoid other limitations and also prevent need for callbacks at frame change - wich you should definitely avoid for stability reasons.

**Reply by alberoman** [reply]
Thank you very much, using the image sequence in the texture node solved the issue perfectly!
